buildSrc/src/main/java/Dependencies.kt

```diff
@@ -13,24 +13,24 @@ object Versions {
     const val osslicenses_library = "17.0.0"
     const val detekt = "1.0.0-RC16"
 
-    const val androidx_appcompat = "1.1.0"
+    const val androidx_appcompat = "1.2.0-alpha02"
     const val androidx_biometric = "1.0.1"
-    const val androidx_coordinator_layout = "1.1.0-beta01"
-    const val androidx_constraint_layout = "2.0.0-beta2"
+    const val androidx_coordinator_layout = "1.1.0-rc01"
+    const val androidx_constraint_layout = "2.0.0-beta4"
     const val androidx_preference = "1.1.0"
     const val androidx_legacy = "1.0.0"
     const val androidx_annotation = "1.1.0"
-    const val androidx_lifecycle = "2.2.0-rc02"
-    const val androidx_fragment = "1.2.0-beta02"
-    const val androidx_navigation = "2.2.0-beta01"
-    const val androidx_recyclerview = "1.1.0-beta04"
+    const val androidx_lifecycle = "2.2.0"
+    const val androidx_fragment = "1.2.1"
+    const val androidx_navigation = "2.3.0-alpha01"
```

Comment:
Do we want to keep using alpha versions or start switching to stable?

Reply:
Yeah this can be 2.2.1 actually